packages/link-modules/src/definitions/product-variant-price-set.ts

```diff
@@ -21,6 +21,8 @@ export const ProductVariantPriceSet: ModuleJoinerConfig = {
   relationships: [
     {
       serviceName: Modules.PRODUCT,
+      // TODO: Remove this when product module is the default product service
+      isInternalService: true,
```

Comment:
question: what does this do?

Reply:
This is to load it from the core and not as being a module.

Reply:
We need that if the product module is not required to run the pricing module, in that case the pricing modue is linked to the internal core service and the pivot table is managed in the core